tests/cypress/views/welcome.js

```diff
@@ -1,287 +0,0 @@
-/** *****************************************************************************
- * Licensed Materials - Property of Red Hat, Inc.
- * Copyright (c) 2020 Red Hat, Inc.
- ****************************************************************************** */
-
```

Comment:
We no longer maintain any tests related to the welcome page. Our tests are only focused on `Search` or `Overview` pages; therefore, I am removing this file since it is dead code.